uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -643,14 +730,20 @@ sub registerScanIntoDB {
         ########################################################
         # register into the db fixme if I ever want a dry run ## 
         ########################################################
-        print "Registering file into db\n" if $this->{debug};
+        if ($this->{verbose}){
+            $message = "\n Registering file into database\n";
+            $this->spool($message, 'N', $upload_id);
+        }  
         $fileID = &NeuroDB::MRI::register_db($minc_file);
-        print "FileID: $fileID\n" if $this->{debug}
+        if ($this->{verbose}){
+            $message = "\n FileID: $fileID\n";
+            $this->spool($message, 'N', $upload_id);
+        }  
 
         ########################################################
         ### update mri_acquisition_dates table #################
         ########################################################
-        &$this->update_mri_acquisition_dates(
+        $this->update_mri_acquisition_dates(
```

Comment:
Why would you want to put the & back ? I think $this->update_mri_acquisition_dates(...) is the appropriate way to call method update_mri_acquisition_dates on object $this.